qiskit/qpy/__init__.py

```diff
@@ -17,13 +17,12 @@
 
 .. currentmodule:: qiskit.qpy
 
-QPY is a binary serialization format for :class:`~.QuantumCircuit` and
-:class:`~.ScheduleBlock` objects that is designed to be cross-platform,
-Python version agnostic, and backwards compatible moving forward. QPY should
-be used if you need a mechanism to save or copy between systems a
-:class:`~.QuantumCircuit` or :class:`~.ScheduleBlock` that preserves the full
-Qiskit object structure (except for custom attributes defined outside of
-Qiskit code). This differs from other serialization formats like
+QPY is a binary serialization format for :class:`~.QuantumCircuit`
+objects that is designed to be cross-platform,  Python version agnostic,
```

Comment:
Super tiny nit, not worth blocking over:
```suggestion
objects that is designed to be cross-platform, Python version agnostic,
```

Reply:
Will apply this in #13662